creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -527,8 +525,8 @@ class SnapbackSM {
         // If for some reason any node in the new replica set is not registered on chain as a valid SP and is
         // selected as part of the new replica set, do not issue reconfig
         if (!this.peerSetManager.endpointToSPIdMap[endpt]) {
+          response.errorMsg = `[issueUpdateReplicaSetOp] userId=${userId} wallet=${wallet} phase=${phase} unable to find valid SPs from new replica set=[${newReplicaSetEndpoints}] | new replica set spIds=[${newReplicaSetSPIds}] | reconfig type=[${reconfigType}] | endpointToSPIdMap=${this.peerSetManager.endpointToSPIdMap} | endpt=${endpt}. Skipping reconfig.`
```

Comment:
can you confirm if this log works as expected - i think we may need to json.stringify the map?

Reply:
yep definitely need to do that. putting in a new PR